src/arch/aarch64/mod.rs

```diff
@@ -155,67 +154,67 @@ pub fn virt_to_phys(
 	Ok(pte.address())
 }
 
-pub fn init_guest_mem(mem: &mut [u8]) {
+pub fn init_guest_mem(mem: &mut [u8], _guest_address: u64) {
 	let mem_addr = std::ptr::addr_of_mut!(mem[0]);
 
-	assert!(mem.len() >= BOOT_PGT.as_u64() as usize + 512 * size_of::<u64>());
+	assert!(mem.len() >= PGT_OFFSET as usize + 512 * size_of::<u64>());
```

Comment:
Given that this part is 'hard to get right' without testing and will most likely not work (despite the function being close to identical with its `x86_64` counterpart) - yet we still need to have a `guest_address` variable (given that it is required in `x86_64`), this variable is deliberately ignored.